app/models/job.rb

```diff
@@ -19,4 +20,8 @@ def assign_tech
     tech_matches = Technology.where(name: raw_technologies)
     self.technologies = tech_matches
   end
+
+  def self.by_tech(tech_name)
+    joins(:technologies).where(technologies: {name: tech_name})
```

Comment:
Interesting. Can you make a scope that accepts an argument? I thought that's why you did it this way/